assets/styles/themes/light.css

```diff
@@ -55,9 +58,9 @@ a, .breadcrumb-item.active a, .dropdown-menu i, .navigation-block i, .autocomple
     background-color: var(--primary-color-dark);
 }
 
-/** PRIMARY COLOR LIGHT **/
+/** PRIMARY COLOR **/
 .header-logo {
-    background-color: var(--primary-color-light);
+    background-color: var(--primary-color);
```

Comment:
Wouldn't it be better if logo had a bit lighter background than the side menu like in the original version (I mean reverting --primary-color-**light**)? There is the same on PG website:
![image](https://github.com/museum-engineering-project/koillectionPG/assets/86925036/f72a91b9-6b0c-4767-a35a-fea8f052f828)

Reply:
Done